doc/index.md

```diff
@@ -91,23 +91,21 @@ interface ISdkConfig {
     logger?: ILogger;
     optOutOfTelemetry?: boolean;
     allowedSessionTypes?: SessionTypes[];
-    media?: {
-        monitorMicVolume?: boolean;
-    };
     defaults?: {
-        audioStream?: MediaStream;
-        audioElement?: HTMLAudioElement;
-        videoElement?: HTMLVideoElement;
-        videoResolution?: {
-            width: ConstrainULong;
-            height: ConstrainULong;
-        };
-        videoDeviceId?: string | null;
-        audioDeviceId?: string | null;
-        micAutoGainControl?: ConstrainBoolean | boolean;
-        micEchoCancellation?: ConstrainBoolean | boolean;
-        micNoiseSuppression?: ConstrainBoolean | boolean;
-        outputDeviceId?: string | null;
+      audioStream?: MediaStream;
+      audioElement?: HTMLAudioElement;
+      videoElement?: HTMLVideoElement;
+      videoResolution?: {
+        width: ConstrainULong;
+        height: ConstrainULong;
+      };
+      videoDeviceId?: string | null;
+      audioDeviceId?: string | null;
+      outputDeviceId?: string | null;
+      micAutoGainControl?: ConstrainBoolean;
+      micEchoCancellation?: ConstrainBoolean;
+      micNoiseSuppression?: ConstrainBoolean;
+      monitorMicVolume?: boolean;
```

Comment:
Just indentation and re-ordering changes. And I moved `monitorMicVolume` in here because I missed that in my last PR.